semantic.py

```diff
@@ -381,16 +394,143 @@ def is_valid_program(self, tree):
 
     # Takes in a measure, builds a list of signals
     def measure_to_signal(self, tree):
-        pass
+        if tree.data != 'measure':
```

Comment:
call isvalidmeasure

Reply:
None is the isvalid stuff is in this branch and will be sorted later